cognite/client/data_classes/hosted_extractors/sources.py

```diff
@@ -821,16 +822,20 @@ def __init__(
 
     @classmethod
     def _load_source(cls, resource: dict[str, Any]) -> Self:
-        return cls(
+        # Using args to avoid repeating the default value for 'scheme'
+        args = dict(
             external_id=resource["externalId"],
             host=resource["host"],
-            scheme=resource["scheme"],
             port=resource.get("port"),
             ca_certificate=CACertificateWrite._load(resource["caCertificate"]) if "caCertificate" in resource else None,
             auth_certificate=AuthCertificateWrite._load(resource["authCertificate"])
             if "authCertificate" in resource
             else None,
         )
+        if "schema" in resource:
+            args["scheme"] = resource["scheme"]
```

Comment:
Shouldn't it be like this? Let's add a test.
```suggestion
            args["scheme"] = resource["schema"]
```